gutenberg.php

```diff
@@ -529,14 +529,18 @@ function gutenberg_replace_default_add_new_button() {
 			button.remove();
 
 			var expander = document.getElementById( 'split-page-title-action' ).getElementsByClassName( 'expander' ).item( 0 );
+			var dropdown = expander.parentNode.getElementsByClassName( 'dropdown' ).item( 0 );
```

Comment:
If we're refactoring, this looks ripe to be simplified:

```js
var dropdown = expander.parentNode.querySelector( '.dropdown' );
```

Reply:
Oh, absolutely! 👍 Fixed.